agent/hibinit-agent

```diff
@@ -415,20 +418,25 @@ def hibernation_enabled(state_dir):
     os.mknod(hib_sem_file)
     return True
 
+def calc_target_swap_size(ram_bytes: int, swap_percentage: int, swap_mb: int) -> int: 
```

Comment:
Type Hints were added to python in version 3.5 | [PEP 484](https://peps.python.org/pep-0484/)

AL2 still uses Python 2.7 and we wan't to avoid different branches for versions of AL. Though this is great in terms of coding style, please avoid using this for backwards compatibility.